### ISSUE TITLE
Postman crashes and deletes itself after installation

### ISSUE THREAD

**Issue by rotispesial** [a85/POSTMan-Chrome-Extension#7716]
**Description**
After opening the zip file for the installer, 

**To Reproduce**
1. Open the zip file (in this case Postman-osx-7.11.0.zip)
2. Open Postman app
3. Confirm Mac that file is safe
4. Open Postman app from dock
5. Confirm to move Postman to application folder
6. App crashes, removes itself from Application folder

**Expected behavior**
Clicking on the Postman app icon should open the app, after all the permissions. Moreover, the app places itself in the Applications folder already, even though the zip file was somewhere else. The expected behavior is the app should open in the same folder as the zip file.

**Screenshots**
Cannot upload video recording :(

**App information (please complete the following information):**
 - App Type - Chrome Browser
 - Postman Version - 7.11.0
 - OS: [e.g. macOS Mojave 10.14.6]

**Additional context**
Resolved this issue by selecting "No. Keep here." when system asks to move the app to Applications folder. App opens after that.

**Comment by harryi3t** [comment]
@kit-teguh Thanks for raising the issue. We would need some more information to help us triage the issue.

1. When did this start happening? Was it a fresh install or an update?
2. If you manually move the app to the `Application` folder, does this happen?
3. Have you downloaded the Postman zip file from our website or transferred it from some USB drive?

**Comment by rotispesial** [comment]
Hi Harendra,

As I mentioned, I found a work around by not accepting the OS to
automatically move Postman to the Applications folder (as the program
already exists in the Application).

To answer your questions:

1. Fresh install
2. As I mentioned, the file opens correctly already in the Applications
folder, but the system still requests to move Postman to the Applications
folder
3. I downloaded the zip file from the website.

I hope this helps!

Cheers,

*Christopher (Kit) Teguh*
Email: kit.teguh@gmail.com
Whatsapp: +855 89 963 110
Phone: +60 17 688 2819

**Comment by flying-turtle-tech** [comment]
I have this issue too. I noticed it last Friday Jan. 24th 2020 and postman app had auto-updated January 16th at 4:14am. Seems to be only an issue after this update from Jan 16.
I was able to use it on January 16th during my work day (after the update) and Jan 24. is the next time I tried to use the app again. 
This is the icon I see on my dock for postman in the applications folder
<img width="75" alt="Screen Shot 2020-01-27 at 2 30 50 PM" src="https://user-images.githubusercontent.com/60302946/73206815-9f3c4c80-4111-11ea-86ea-30af2cc11fb1.png">

This is the initial error message I saw on Jan 24.
<img width="418" alt="Screen Shot 2020-01-24 at 4 27 02 PM" src="https://user-images.githubusercontent.com/60302946/73206923-db6fad00-4111-11ea-984f-3046b9691e8c.png">

Tried to use the app again today and it did not work. Moved the application to Trash and removed from dock. Downloaded from https://www.getpostman.com/downloads/. When I let Postman move itself to the Applications, or I move it manually, I get an instant crash. I also don't see the postman logo for the app (same as first image above).

If I keep it in the location I initially saved it to the app works fine.

macOS Catalina Version 10.15.1
This is issue is on the macOSX app

**Comment by DannyDainton** [comment]
Hey @kit-teguh / @jonathan-bos 

Is this still an issue with the latest versions of the app for you?

**Comment by DannyDainton** [comment]
I'm going to close this issue but we can open this up again if this is still a problem that people are facing.

**Comment by wischi-chr** [comment]
I'm not sure if my problem was the same (I found this issue after googling _"postman uninstalls itself"_). My PC is basically always on and it looks like that postman uninstalled itself (worked fine yesterday - or at least before the weekend).

Are there any logs I can provide which would help you diagnose the problem?

**Update:**
A bit more details about my setup because it might play a part:
I use chocolatey and have the package https://chocolatey.org/packages/postman installed. A nightly cron job automatically tries to upgrade everything (except a hand full of version-pinned packages) with `choco upgrade all`

I'm not sure if the choco package is official or just a community package, but it looks like the last autoupgrade ran the setup for 8.0.3 (`https://dl.pstmn.io/download/version/8.0.3/windows64`) completed "successfully" but only removed the old installation.

I just run the setup manually and it worked. Than I uninstalled it, removed it from choco and installed it again and it also worked.
My guess is that the automatic update with choco from 7.36.1. to 8.0.3 somehow failed.

If the choco package is official maybe a few more tests are needed (like specifically installing an old version first and try to upgrade it to 8.0.3)
